process-analyzer/src/com/axonivy/utils/process/analyzer/managedbean/TimeIntervalFilterBean.java

```diff
@@ -95,6 +75,14 @@ public void onSelectDateTime() {
     unifyFilterAndRefreshData();
   }
 
+  public void onSelectDateRange() {
+    if (CollectionUtils.isNotEmpty(selectedDateRange) && selectedDateRange.size() > 1) {
```

Comment:
I think we don't need the condition. If there's no date selected or only 1 date it will throw error